src/main.cpp

```diff
@@ -184,16 +184,16 @@ main(int argc, char* argv[])
         ? std::optional<std::reference_wrapper<ssl::context>>{ctx.value()}
         : std::nullopt;
 
-    auto const threads = config->contains("workers")
-        ? config->at("workers").as_int64()
-        : std::thread::hardware_concurrency();
+    auto const threads = config->contains("io_threads")
+        ? config->at("io_threads").as_int64()
+        : 2;
```

Comment:
so now `workers` is the number of threads actually processing requests?

Reply:
yes, though I'm not tied to this change. I think these are the field names rippled uses, but we can use something else. What do you think?